### ISSUE TITLE
XFCE--repo package Plank: no dots under active icons

### ISSUE THREAD

**Issue by brent111** [getsolus/packages#1464]
### Summary

Installed Plank. Opened apps. No dots.

### Steps to reproduce

Open plank in terminal, launch any app, there is not dot showing you app is open when minimized.
Open plank in menu, launch any app, there is not dot showing you app is open when minimized.

### Expected result

Minimize an app in Plank. Observe the icon in the dock with the dot under it reminding you it is minimized.

### Actual result

Minimize an app in Plank. Observe the icon in the dock with the with NO dot under it to remind you it is minimized.

### Environment

- [ ] Is system up to date?

### Repo

Shannon (stable)

### Desktop Environment

Budgie

### System details

XFCE, up to date.

### Other comments

THIS BUG IS SOLVED or is it? Up to you to decide.
I have a 1920 1080 _EDIT_ display. Changing my refresh rate from 75 to 60 FIXED this--the dot suddenly became visible. Meaning app somewhat broken OOTB depending on any monitor' Refresh rate.
Thought you'd like to know. 
I am not asking for a fix, I made a workaround.
Not thrilled with the 60 but it will be OK.

**Comment by brent111** [comment]
also if relevant I have newish Acer monitor...75 and 60 my only refresh rates for the GEForce 430, an old card.

**Comment by brent111** [comment]
I don't use Plank in Solus Budgie.
But........in a bizarre addendum........my Endeavour shows no dots at 60..,Solus XFCE can only see plank dots at 60.
Endeavour only shows plank dots at a refresh rate of 75 while solus xfce cannot at 75.

I dont' know if you can use Arch as a control here--it's backwards world with our XFCE and Endeavour Budgie as far as Plank dot visibility and refresh rate.

**Comment by brent111** [comment]
close/reopen here happened when I deleted comment (dots changed again). please disregard.

**Comment by brent111** [comment]
ADDENDUM: CANCEL THIS? PROBLEM **NEITHER XFCE OR PLANK.**

To wit: In Budgie Dock (not plank) at 75 refresh rate, NO DOTS:

![nodots](https://github.com/getsolus/packages/assets/143135556/0fb7659d-058d-4647-bc8c-c8ec94a1a8e6)

Now at 60 refresh, budgie dock, not Plank, I have dots:

![dots](https://github.com/getsolus/packages/assets/143135556/df9c0caf-a481-4d35-891c-4349b9c176ef)


CONCLUSION: I was at 75 when I got the screen months ago. Somewhere along the line Solus introduced something that has forced my hand to 60 with no other alternative (for seeing Plank or Panel dots.)

I hate the weird ones:).
Flush this post Bug Issue Ticket. Experiments in Budge have made my issue no longer valid.

**Comment by ermo** [comment]
> Flush this post Bug Issue Ticket. Experiments in Budge have made my issue no longer valid.

Closing as invalid per author's wishes.

**Comment by brent111** [comment]
@ermo I was in over my head. I mean in the end, for Plank on XFCE, it became positional. I had to accept I would get no dots on top. In the end I got them on the bottom. And if I hung a dock on the right they'd be barely visible. All this with a black background. And this is with redshift off. New screen, centered on it's X/Y. 

Refresh rates as a solution _did not stick_ so I did not report back.
In the end it was fixed in Budgie, but not XFCE but for some reason not convinced this was XFCE problem as I had to make some moves in its compositor.

The prudent thing was to close this, thank you.
